diff --git a/src/util/syscall_sandbox.cpp b/src/util/syscall_sandbox.cpp
index a05efac602..4809ceb250 100644
--- a/src/util/syscall_sandbox.cpp
+++ b/src/util/syscall_sandbox.cpp
@@ -867,6 +867,8 @@ void SetSyscallSandboxPolicy(SyscallSandboxPolicy syscall_policy)
         break;
     case SyscallSandboxPolicy::MESSAGE_HANDLER: // Thread: msghand
         seccomp_policy_builder.AllowFileSystem();
+        // ELEMENTS: Need network to call CallMainChainRPC
+        seccomp_policy_builder.AllowNetwork();
         break;
     case SyscallSandboxPolicy::NET: // Thread: net
         seccomp_policy_builder.AllowFileSystem();
